<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAM2ascii</class>
 <widget class="QMainWindow" name="CAM2ascii">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">CAM2ascii</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="loadingMenuFrame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	
	background-color: rgb(47, 47, 47);
	color: white
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>1281</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>$&gt; _</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>460</y>
       <width>720</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	border-radius : 10px;
	border-style: none;
	background-color: rgb(80, 80, 80);
	color:white;
	text-align: center;
}

QProgressBar::chunk {
	border-radius : 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.512, x2:1, y2:0.579227, stop:0 rgba(0, 153, 102, 255), stop:1 rgba(0, 104, 69, 255));
}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="author">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>700</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color:white
}</string>
     </property>
     <property name="text">
      <string>Made by Eppah</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
